pkg/kubelet/cm/dra/plugin/plugins_store.go

```diff
@@ -88,31 +41,22 @@ func (s *PluginsStore) Get(pluginName string) *Plugin {
 
 // Set lets you save a DRA Plugin to the list and give it a specific name.
 // This method is protected by a mutex.
-func (s *PluginsStore) Set(pluginName string, plugin *Plugin) {
+func (s *pluginsStore) set(pluginName string, p *plugin) {
 	s.Lock()
 	defer s.Unlock()
 
 	if s.store == nil {
-		s.store = make(map[string]*Plugin)
+		s.store = make(map[string]*plugin)
 	}
 
-	s.store[pluginName] = plugin
+	s.store[pluginName] = p
```

Comment:
Should we check if `pluginName` is already in the store?

Reply:
What do you suggest ? Log a warning, refuse to add the plugin if it already exists or just override the previous as it is done now ?

Reply:
Let's start with a warning. Covering this with e2e would also make sense.

Reply:
sure thing, will do

Reply:
warning added in 17b634683020ed2ab31936ac3248cc8c9cda97b5